docs/openapi/components/schemas/credentials/SIMASteelImportLicenseCredential.yml

```diff
@@ -569,8 +569,8 @@ example: |-
       "countryOfOrigin": "MX",
       "countryOfExportation": "MX",
       "expectedPortOfEntry": "USMOB",
-      "expectedDateOfExport": "2022-04-02T08:00:00Z",
-      "expectedDateOfImport": "2022-04-09T10:30:00Z",
+      "expectedDateOfExport": "2022-04-02",
+      "expectedDateOfImport": "2022-04-09",
```

Comment:
The trouble with making these `date` instead of `date-time` is that timezone falls off, which can make the `date` ambiguous as it changes at midnight — but *which* midnight?